vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -155,6 +162,17 @@ def parse_pod(pod)
         end
       end
 
+      createdby_txt = pod.metadata.annotations.try("kubernetes.io/created-by")
```

Comment:
Wow, yeah, that's kinda crazy.  Am I understanding this correctly, there's a method on the annotations called "kubernetes.io/created-by"?  Where is this defined?  I see some define_method and sends in kubeclient [here](https://github.com/abonas/kubeclient/blob/80b39b14e7ddfde23192f2ececbbb06be15ced00/lib/kubeclient/common.rb#L195) and [here](https://github.com/abonas/kubeclient/blob/80b39b14e7ddfde23192f2ececbbb06be15ced00/lib/kubeclient/common.rb#L38-63) but I can't tell where "annoations" is defined or the "kubernetes.io/created-by".

 It seems problematic because you can't easily access the method directly, you have to "send" it to the object.  

We should link to where this chain of method calls is defined so "future us" can understand it.